<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Prashant_Laptop" failures="1" tests="6" name="Test" time="0.112" errors="0" timestamp="2024-10-25T17:05:53 IST">
  <testcase classname="reRunTestCases.TestCases" name="method1" time="0.024">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="reRunTestCases.TestCases" name="method1" time="0.002">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="reRunTestCases.TestCases" name="method1" time="0.002">
    <skipped/>
  </testcase> <!-- method1 -->
  <testcase classname="reRunTestCases.TestCases" name="method1" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at reRunTestCases.TestCases.method1(TestCases.java:9)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- method1 -->
  <testcase classname="reRunTestCases.TestCases" name="method2" time="0.002"/>
  <testcase classname="reRunTestCases.TestCases" name="method3" time="0.003"/>
</testsuite> <!-- Test -->
